docs/getting_started/contract_manage.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Contract management
+sidebar_position: 12
+---
+
+Contract management is a crucial feature that allows users to effectively interact with installed and deployed contracts. This feature enables users to upload their own `.wasm` and `ABI` files to the Console and deploy them to the blockchain.
+
+When you navigate to the Contract Management Page, you will be presented with two tabs: **Installed Contracts** and **Deployed Contracts**.
+
+![](../../static/img/contract_management.png)
+
+
+## Installed Contract
+
+Installed contract page shows the list of installed contracts for your specific shard. Installing the contract means putting the contract in the Installed state, meaning uploading the contract files to GCS and making the contract available for deployment from the Calimero Console. To install your contract:
+
+- Clicking on the **Install contract** button, which will prompt you to upload your contract files.
```

Comment:
Click on the

remove the . at the end

Reply:
The button is called Install new contract

Reply:
why are we removing the .dot